app/gui2/src/stores/graph/graphDatabase.ts

```diff
@@ -559,10 +546,17 @@ export class GraphDb {
   }
 }
 
-interface NodeInfo {
+/** Source code data of the specific node. */
+interface NodeSource {
+  /** The outer AST of the node (see {@link NodeDataFromAst.outerExpr}). */
```

Comment:
We could add information to `NodeDataFromAst.outerExpr` what it is for input node (it is only arg name? Or the name with default value and type?)